src/gui/scheduling/syncscheduler.cpp

```diff
@@ -143,10 +151,13 @@ void SyncScheduler::startNext()
         connect(
             _currentSync, &Folder::syncFinished, this,
             [this](const SyncResult &result) {
+                qCInfo(lcSyncScheduler) << "Sync finished for" << _currentSync->displayName() << "with status" << result.status();
                 if (result.status() != SyncResult::Success) {
                     // Retry a couple of times after failure; or regularly if requested
-                    if (_currentSync->canSync() && (_currentSync->consecutiveFailingSyncs() > 0 && _currentSync->consecutiveFailingSyncs() < 3)
+                    if ((_currentSync->canSync() && (_currentSync->consecutiveFailingSyncs() > 0 && _currentSync->consecutiveFailingSyncs() < 3))
                         || _currentSync->syncEngine().isAnotherSyncNeeded() == AnotherSyncNeeded::DelayedFollowUp) {
+                        qCInfo(lcSyncScheduler) << "Rescheduling sync for" << _currentSync->displayName() << "in" << SyncEngine::minimumFileAgeForUpload
```

Comment:
We have a pretty printer for std::chrono.
`23-09-22 10:07:27:153 [ info scheduler.syncscheduler ]:	Rescheduling sync for "Personal" in duration(0h, 0min, 2s, 0ms) seconds`
As the delay is well, not very important we can just omit it from the log?

Reply:
will do, wasn't sure about that.